<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuppressAlarmDialog</class>
 <widget class="QWidget" name="SuppressAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inhibit System / Non-physical Subsystem Alarm Dialog - [AssetName].[AlarmType]</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="CtlRadioButton" name="rdoByAlarmType">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Inhibits all system alarms and non-physical subsystem alarms of this Asset and Alarm Type.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CtlRadioButton" name="rdoByDescription">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Inhibits all system alarms and non-physical subsystem alarms of this Description for the Asset and Alarm Type.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CtlButton" name="btnOk">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CtlButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>CtlButton</class>
      <extends></extends>
      <header>bus/common_control/src/button/CtlButton.h</header>
    </customwidget>
    <customwidget>
      <class>CtlRadioButton</class>
      <extends></extends>
      <header>bus/common_control/src/button/CtlRadioButton.h</header>
    </customwidget>
  </customwidgets>
 <resources/>
 <connections/>
</ui>
